xml_inscriptions/transcribed/PA.Phil.UP.UM.L.MS3451

```diff
@@ -0,0 +1,214 @@
+                                via funds from <persName role="donor">Lucy Wharton Drexel</persName> in <date when="1897">1897</date></p>
+                        </provenance>
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ref type="external" target="https://www.penn.museum/collections/object/119540">University of Pennsylvania Museum web entry</ref>
+                    </bibl>
+                    <bibl>
+                        <ref type="external" target="www.trismegistos.org/text/547798">TM547798</ref>
+                    </bibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#EphEp_IX"/>
+                        <biblScope unit="entry">649</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Bodel_Tracy"/>
+                        <biblScope unit="page">212</biblScope>
+                    </bibl>
+                    <bibl> 
+                        <ptr type="rest-of-bibl" target="#AE_1999"/>
+                        <biblScope unit="entry">416</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2025-03-13" who="Mac Carley">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="PA.Phil.UP.UM.L.MS3451_a.jpg"/>
+        </surface>
+        <surface>
+            <desc/>
+            <graphic url="PA.Phil.UP.UM.L.MS3451_b.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>Face A: Chio, Face B: D(onum), Face C: D(edit)</ab></div>
```

Comment:
Different faces need to be listed as different `<div type="textpart">` elements; also remember to do the markup for the abbreviation expansions.